### PR TITLE
Issue 3155 fix

### PR DESCRIPTION
**Pull request by rashmi73**

### CODE REVIEW

**Comment by maxceem** [comment]
@rashmi73 current PR doesn't fix the issue. Would be able to fix it or should I open this issue for pick up?

**Comment by rashmi73** [comment]
@maxceem sorry for delay I would be able to fix it

**Comment by rashmi73** [comment]
@maxceem can we clear everything after successful invites? so it would just endup to making a single line change and not much difficulty

**Comment by rashmi73** [comment]
@maxceem PR update

File: src/components/TeamManagement/ProjectManagementDialog.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ class ProjectManagementDialog extends React.Component {
     if (processingInvites && !nextProps.processingInvites ) {
       const notInvitedSelectedMembers = _.reject(selectedMembers, (selectedMember) => (
         this.isSelectedMemberAlreadyInvited(nextProps.projectTeamInvites, selectedMember)
+        || this.isSelectedMemberAlreadyInvited(nextProps.topcoderTeamInvites, selectedMember)
```
Comment (maxceem) [inline src/components/TeamManagement/ProjectManagementDialog.js]: @rashmi73 is this change required for the fix?

File: src/helpers/utils.js [hunk 2]
```diff
@@ -53,9 +53,12 @@ export const compareEmail = (email1, email2, options = { UNIQUE_GMAIL_VALIDATION
  *
  * @returns {Boolean} true if two user handles are same
  */
-export const compareHandles = (handle1, handle2) => (
-  (handle1 || '').toLowerCase() === (handle2 || '').toLowerCase()
-)
+export const compareHandles = (handle1, handle2) => {
+  if (handle2.indexOf('@') === 0) {
+    return (handle1 || '').toLowerCase() === (handle2.substring(1) || '').toLowerCase()
```
Comment (maxceem) [inline src/helpers/utils.js]: @rashmi73 as before we have logic like this `(handle2 || '')` it looks like sometimes `handle2` can be undefined. To be sure that this wouldn't cause error in runtime it's better to apply `.substring(1)` not to `handle2` directly, but first, we make sure that we have string, like this:
```js
(handle2 || '').substring(1).toLowerCase()
```

File: src/components/TeamManagement/ProjectManagementDialog.js (hunk 1 above)
Comment (rashmi73) [reply src/components/TeamManagement/ProjectManagementDialog.js]: @maxceem yes because may be he is not a customer, because i saw parent component where props are declared, invitest filtered by 'customer' so it may fall into another array? which is topcoderTeamInvites.

File: src/helpers/utils.js (hunk 2 above)
Comment (rashmi73) [reply src/helpers/utils.js]: @maxceem sure updating now

**Comment by rashmi73** [comment]
@maxceem updated PR

File: src/helpers/utils.js [hunk 3]
```diff
@@ -53,9 +53,17 @@ export const compareEmail = (email1, email2, options = { UNIQUE_GMAIL_VALIDATION
...
+export const compareHandles = (handle1, handle2) => {
+  let h1 = handle1
+  let h2 = handle2
+  if (handle2.indexOf('@') === 0) {
+    h2 = (handle2 || '').substring(1)
+  }
+  if (handle1.indexOf('@') === 0) {
+    h1 = (handle1 || '').substring(1)
+  }
+  return h1.toLowerCase() === h2.toLowerCase()
```
[the remark this answers is not in the document]
Comment (rashmi73) [reply src/helpers/utils.js]: @maxceem updated

File: src/components/TeamManagement/ProjectManagementDialog.js (hunk 1 above)
Comment (rashmi73) [reply src/components/TeamManagement/ProjectManagementDialog.js]: @maxceem updated

**Comment by rashmi73** [comment]
@maxceem PR update. Hope it works

**Comment by rashmi73** [comment]
@maxceem done and updated
